apps/nextjs-website/src/_contents/makeDocs.ts

```diff
@@ -107,3 +113,19 @@ export const makeGuide = ({
     parseDocOrThrow
   );
 };
+
+export const makeSolution = ({ solution, bannerLinks }: SolutionDefinition) => {
+  const solutionPath = `/solutions/${solution.slug}`;
```

Comment:
As you can see in video i got an error while navigating inside solutions detail, the problem is that is mounted a wrong html. 
[Screencast from 27-06-2024 10:37:27.webm](https://github.com/pagopa/developer-portal/assets/24475319/02641dca-de05-4c1c-9133-158b94edfd50)

One of the issue i think is due to path prefix.
```suggestion
  const solutionPath = `/solutions/${solution.slug}/details`;
```
But is not enough to fix the problem. As in guide and tutorial route should be different! I think solution detail page path should be something like `apps/nextjs-website/src/app/solutions/[solutionSlug]/details/[...solutionDetailPage]/page.tsx` and maybe some more fixes are needed to manage redirect and fragment properly :thinking: